_episodes/02-installation.md

```diff
@@ -1,119 +1,192 @@
-{: .callout}
+In this tutorial we will be using the [Conda](https://conda.io/projects/conda/en/latest/index.html)
+package manager to install the ESMValTool.
+Other installation methods are also available, they can be found in the
+[documentation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html).
+ESMValTool also contains diagnostics written in [Julia](https://julialang.org/).
+Because Julia cannot be installed by Conda, we will install Julia separately.
+We will first learn how to install Conda, Julia, and finally the ESMValTool.
+We end this chapter by testing that the installation was successful.
 
-## Obtaining the source code
+## Install Conda
 
-The ESMValTool source code is available on a public GitHub repository:
-https://github.com/ESMValGroup/ESMValTool
+ESMValTool is distributed using [Conda](https://conda.io/). We will be using the Miniconda minimal installer for conda. We suggest a Python 3 based installer, though if you happen to already have Conda installed it should also work with Python 2. For more information about installing conda, see [the conda installation documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
 
-The easiest way to obtain it is to clone the repository using git. 
-To clone the public repository open a terminal window and type:
+### Linux
 
-~~~
-    git clone https://github.com/ESMValGroup/ESMValTool.git
-~~~
-{: .source}
+1.  Please download Miniconda3 for Linux at [the miniconda page](https://docs.conda.io/en/latest/miniconda.html). The 64 bit version should work on any recent system. If you have problems in the next step(s) you can alternatively try a 32 bit version.
 
-By default, this command will create a folder called ESMValTool containing the
-source code of the tool. 
+2.  Next, run the installer
 
-Move into the directory to continue installation. 
+    ```bash
+    bash Miniconda3-latest-Linux-x86_64.sh
+    ```
 
-> ## Attention
-> 
-> Make sure that the master branch is checked out before continuing intallation:
-> ~~~
-> git checkout master
-> ~~~ 
-> {: .source}
-{: .callout}
+3.  Follow the instructions in the installer. The defaults should normally suffice.
 
-## Prerequisites
+4.  You will need to restart your terminal for the changes to have effect.
 
-It is recommended to use conda to manage ESMValTool dependencies.
-For a minimal conda installation go to https://conda.io/miniconda.html. To
-simplify the installation process, an environment definition file is provided
-in the repository (``environment.yml`` in the root folder).
+5.  Verify you have a working conda installation by listing all installed packages
 
-> ## Attention
-> It is preferable to use a local, fully user-controlled conda installation.
-> If you have conda installed already, make sure it is up to date by running ``conda update -n base conda``.
-{: .callout}
+    ```bash
+    conda list
```

Comment:
It might be nice to add an 'output' code block showing what it should look like if the installation was successful.

Reply:
Output added. It was clipped using `.,..` and looks OK to me.